hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_KL27Z/drivers/fsl_spi_dma.c

```diff
@@ -168,7 +168,7 @@ void SPI_MasterTransferCreateHandleDMA(SPI_Type *base,
     handle->bytesPerFrame = 1U;
 #endif /* FSL_FEATURE_SPI_16BIT_TRANSFERS */
 
-#if defined(FSL_FEATURE_SPI_FIFO_SIZE) && (FSL_FEATURE_SPI_FIFO_SIZE > 1)
+#if defined(FSL_FEATURE_SPI_HAS_FIFO) && (FSL_FEATURE_SPI_HAS_FIFO)
```

Comment:
The first check is not needed http://stackoverflow.com/questions/5085392/what-is-the-value-of-an-undefined-constant-used-in-if